### PR TITLE
Feat: Netlify Logs quickstart

### PR DESCRIPTION
**Pull request by NetlifyMatt**
New files for the Netlify Logs quickstart and updates for the Netlify Builds quickstart (name updates and associated files).

# Summary

Creating new files for the Netlify Logs quickstart and updating the name and file structure for the existing Netlify Builds quickstart.

## Pre merge checklist

- [x] Did you check you NRQL syntax? - Does it work?
- [x] Did you check your dashboard image quality? -  Do they look good?
- [ ] Did you check that your alerts actually work?
- [x] Did you include an InstallPlan and Documentation reference?
- [x] Did you check your descriptive content for voice, tone, spelling and grammar errors?
- [x] Did you attach images of your dashboards to the PR so we can see them working?

### Screenshots

Attach images of any visual changes, such as a dashboard here.

netlify-logs01
netlify-logs02

### CODE REVIEW

**Comment by jpvajda** [comment]
This looks to have a conflict in `quickstarts/netlify/config.yml` can you take a look at that @NetlifyMatt or @jcountsNR

File: quickstarts/netlify/netlify-logs/config.yml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  - serverless functions
+  - newrelic partner
+
+# Reference to install plans located under /install directory
+# Allows us to construct reusable "install plans" and just use their ID in the quickstart config
+installPlans:
+  - third-party-netlify-logs
+
+documentation:
+  - name: Configuration Docs
```
Comment (jpvajda) [inline quickstarts/netlify/netlify-logs/config.yml:63]:
```suggestion
  - name: Configuration docs
```

**Review by jpvajda — CHANGES_REQUESTED** [review]

**Comment by aaronbassett** [comment]
Hey folks I'm trying to get PR #932 reverted as it deleted some images which are required by the Netlify quickstart. This will probably impact this PR as well as the directory structure has changed.

Also, the change in directory structure will mean that the [image URL in this markdown widget](https://github.com/newrelic/newrelic-quickstarts/blob/c17ef1252ea427c88e4592a5b446abce1bcdc1c9/quickstarts/netlify/dashboards/netlify.json#L70) will need to be updated too

File: quickstarts/netlify/netlify-builds/config.yml [hunk 2]
```diff
@@ -3,15 +3,15 @@ id: 5861d7f5-26c0-43ad-bda8-c893c4b27b25
 name: netlify builds
 
 # Displayed in the UI (required)
-title: Netlify builds
```
[the remark this answers is not in the document]
Comment (NetlifyMatt) [reply quickstarts/netlify/netlify-builds/config.yml:6]: Sentence case is fine.

File: quickstarts/netlify/netlify-builds/config.yml [hunk 3]
```diff
@@ -3,15 +3,15 @@ id: 5861d7f5-26c0-43ad-bda8-c893c4b27b25
...
+title: Netlify Builds
 
 # Long-form description of the quickstart (required)
 description: |
   Netlify is an all-in-one platform for automating modern web projects. Replace your hosting infrastructure, continuous integration, and deployment pipeline with a single workflow. Integrate dynamic functionality like serverless functions, user authentication, and form handling as your projects grow.
 
-  ### Netlify quickstart highlights
+  ### Netlify Builds quickstart highlights
```
[the remark this answers is not in the document]
Comment (NetlifyMatt) [reply quickstarts/netlify/netlify-builds/config.yml:12]: Not a proper name; sentence case OK.

**Comment by jcountsNR** [comment]
@jpvajda  done. It looks like the only conflict was on the name which reverted. @aaronbassett , the link should still work since there the netlify/images dir is still correct.

**Review by jpvajda — APPROVED** [review]

**Comment by jpvajda** [comment]
closing for https://github.com/newrelic/newrelic-quickstarts/pull/967

**Pull request closed without merging**